Zend/zend_vm_gen.php

```diff
@@ -19,7 +19,7 @@
 	 $Id$
 */
 
-$header_text = <<< DATA
+const HEADER_TEXT = <<< DATA
```

Comment:
you are right, I somehow got the impression this makes it 5.6+. Sorry for the noise.